src/SchemaDesign/SchemaDesignerQuestion.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable dot-notation */
+import { SchemaQuestion } from '@/types/interfaces'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import TextField from '@mui/material/TextField'
+import React from 'react'
+
+export default function SchemaDesignerQuestion({
+  onSubmit,
+  question,
+  closeQuestionPicker,
+}: {
+  onSubmit: (data: SchemaQuestion) => void
+  question: any // type this
+  closeQuestionPicker: () => void
+}) {
+  const [formInput, setFormInput] = React.useState<SchemaQuestion>({
+    title: '',
+    description: '',
+    type: question.type,
+    reference: '',
+    format: question.format || '',
+  })
+
+  const handleChange = (event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>) => {
+    const { value, name }: { value: string | number; name: string } = event.target
+    if (value || (!value && event.target.attributes['required'])) {
```

Comment:
Something's not right here, we shouldn't be disabling eslint to get around this type issue. It looks to me like you've typed `event` correctly. Are you sure `event.target.attributes.required` exists?

Happy to look into this further on Monday, but do not have time this evening.